### PR TITLE
cart-page added and routing added for cart-page

### PR DESCRIPTION
**Pull request by rishabh486**
pr request created for cart page

### CODE REVIEW

**Review by humzahasan — APPROVED** [review]
- Good file strucutre
- Good naming convention followed

Good to merge!

File: src/Components/cart/cart.css [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  }
+  .card-margin {
+    margin: 2rem;
+  }
+  .card-horizontal {
+    display: flex;
+    flex-direction: row;
+    width: 55rem;
+    max-height: 31rem;
+    box-shadow: 0 0 20px rgba(0, 0, 0, 0.2);
```
Comment (sumitk-singh) [inline src/Components/cart/cart.css:15]: use color variables

Comment (zoef-yt) [inline src/Components/cart/cart.css:15]: use var for color or make a box-shadow var so you can be consistent across the whole app.

File: src/Components/cards/card.jsx [hunk 2]
```diff
@@ -1,10 +1,12 @@
 import React from "react";
```
Comment (zoef-yt) [inline src/Components/cards/card.jsx:1]: image
Your cards are overflowing from containing them in a div should fix the issue..

File: src/Components/navbar/navbar.jsx [hunk 3]
```diff
@@ -20,11 +23,11 @@ function Navbar() {
                         <div class="icon-badge">
                             <button type="button" class="icon-button" id="cart-button">
                                 <span>
-                                  <img class="icon-img" src="/Images/icons8-trolley-cart-64.png" />
+                                <Link to="/cart-page"> <img class="icon-img" src="/Images/icons8-trolley-cart-64.png" /></Link>
```
Comment (zoef-yt) [inline src/Components/navbar/navbar.jsx:26]: image
Image is not being loaded.

File: src/context/cart-context.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
+import { useContext, createContext, useReducer } from "react";
+import { useProduct } from "./product-context";
+import { cartReducer } from "../reducers/cart-reducer";
+const CartContext = createContext();
+const CartProvider = ({ children }) => {
+  const { items } = useProduct();
+  const [state, dispatch] = useReducer(cartReducer, {
```
Comment (sumitk-singh) [inline src/context/cart-context.js:7]: `useReducer`  has used in very clean way, good job.

File: src/Components/navbar/navbar.jsx (hunk 3 above)
Comment (zoef-yt) [inline src/Components/navbar/navbar.jsx:26]: Add alt tag to img

File: src/Components/navbar/navbar.jsx [hunk 5]
```diff
@@ -20,11 +23,11 @@ function Navbar() {
                         <div class="icon-badge">
```
Comment (zoef-yt) [inline src/Components/navbar/navbar.jsx:23]: Also you are using anchor tags for home login and sign use 'Link' or 'Navlink' as there's no index.html or login.html file.

File: src/App.js [hunk 6]
```diff
@@ -1,10 +1,14 @@
 import "./App.css";
```
Comment (zoef-yt) [inline src/App.js:1]: image
Add _redirect file too

**Review by zoef-yt — CHANGES_REQUESTED** [review]

**Review by sumitk-singh — APPROVED** [review]
overall codes look clean and easy to read,
Good to go.

**Pull request merged**